week1/3-website/css/style.css

```diff
@@ -6,3 +6,176 @@
  *    for example: General styles, Navigation styles, Hero styles, Footer etc.
  * 
  */
+
+body {
+    margin: 20px;
+    line-height: 1.5; /* to create gaps between lines for easy readying*/
+}
+
+ header, footer {
+     color: rgb(17, 80, 17);
+ }
+ 
+ /*BUG: I tried header:not(ul) to select everything in header but not ul but it didn't work*/
+ h1, h2, footer {
+     text-align: center;
+ }
+
+ /*article subheadings*/
+ h3 {
+    color: rgb(30, 139, 30);
+    text-shadow: 4px 4px 6px rgb(80, 60, 17);
+ }
+
+ /*title cooked*/
+ .cooked {
+     text-decoration: line-through;
+     background: rgb(0, 0, 0);
+     color: rgb(255, 255, 255);
+     transform: rotate(-30deg); /*BUG not working - I want to rotate the text in the title*/
+ }
+
+/*BUG: linear-gradient not working*/
+ footer {
+    background: linear-gradient (rgb(0, 255, 0), rgb(255, 255, 255));
```

Comment:
Here you just have an unwanted white space between `linear-gradient` and `(rgb(0, 255...`